src/Makefile.inc

```diff
@@ -37,6 +37,7 @@ CURL_CFILES = \
 	tool_dirhie.c \
 	tool_doswin.c \
 	tool_easysrc.c \
+	tool_filetime.c \
```

Comment:
you forgot to add the new header file as well to Makefile.inc, which caused the "distcheck" build to fail on travis!

Reply:
Thanks, fixed!